webpack/assets/javascripts/react_app/components/common/forms/RadioButtonGroup.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Field } from 'redux-form';
+
+import CommonForm from './CommonForm';
+
+const RadioButtonGroup = ({
+  controlLabel, radios, name, className = '', inputClassName = 'col-md-6',
+}) =>
+  (
+    <CommonForm label={controlLabel} className={className} inputClassName={inputClassName}>
+      {radios.map((item, index) => (
+        <label className='radio-inline' key={index}>
+          <Field name={name} component="input" type="radio" value={item.value} onChange={item.onChange} checked={item.checked} />
+          { item.label }
+        </label>))
+      }
+    </CommonForm>
+  );
+
+RadioButtonGroup.propTypes = {
+  controlLabel: PropTypes.string.isRequired,
+  radios: PropTypes.arrayOf(PropTypes.shape({ value: PropTypes.string })),
```

Comment:
I am not sure but maybe it is a good idea to mention here that each radio has `label`?
WDYT?